legend-engine-xt-flatdata-pure/src/main/resources/core_external_format_flatdata/executionPlan/tests/utils.pure

```diff
@@ -17,9 +17,9 @@ import meta::legend::*;
 import meta::pure::extension::*;
 import meta::pure::runtime::*;
 
-function meta::external::format::flatdata::executionPlan::test::conertStringToByteStream(data:String[1]): InstanceValue[1]
+function meta::external::format::flatdata::executionPlan::test::convertStringToByteArray(data:String[1]): InstanceValue[1]

```

Comment:
I think convertStringToBytes is a better name, since in some contexts we won't be interpreting Byte[*] as an array.
```suggestion
function meta::external::format::flatdata::executionPlan::test::convertStringToBytes(data:String[1]): InstanceValue[1]

```

Reply:
makes sense done